src/pages/u/settings.astro

```diff
@@ -2,12 +2,13 @@
 import LayoutSidebar from '@layouts/LayoutSidebar.astro';
 import UserSettings from '@components/UserSettings.astro';
 import { url, api } from '@lib/helpers';
+import type { User } from '@lib/types';
```

Comment:
Unused import?

Reply:
It was forgotten in an earlier commit i guess.. See line 24/25